THIRD_PARTY_LICENSES_RUST_CRATES.html

```diff
@@ -44,10 +44,11 @@ <h1>Third Party Licenses</h1>
     
         <h2>Overview of licenses:</h2>
         <ul class="licenses-overview">
-            <li><a href="#MIT">MIT License</a> (29)</li>
-            <li><a href="#Apache-2.0">Apache License 2.0</a> (27)</li>
+            <li><a href="#Apache-2.0">Apache License 2.0</a> (36)</li>
+            <li><a href="#MIT">MIT License</a> (31)</li>
+            <li><a href="#BSD-3-Clause">BSD 3-Clause &quot;New&quot; or &quot;Revised&quot; License</a> (4)</li>
             <li><a href="#ISC">ISC License</a> (3)</li>
-            <li><a href="#BSD-3-Clause">BSD 3-Clause &quot;New&quot; or &quot;Revised&quot; License</a> (2)</li>
+            <li><a href="#Unicode-DFS-2016">Unicode License Agreement - Data Files and Software (2016)</a> (1)</li>
```

Comment:
Need to be double-checked form legal perspective

Reply:
This was auto-generated, will look into it.